fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1 +1,11 @@
-The reasonable Tox client for Android.
+<p><b>aTox, a beautiful and modern Tox client.</b></p>
+
+<p>aTox is a free open-source messaging app for the Tox protocol.</p>
+
+<p><b>Why aTox?</b></p>
+<ul>
+<li>Open-source: free to share, inspect, and change as you like.</li>
```

Comment:
```
 <li>Open-source: free to share, inspect, and change as you like.</li>
 <li>Reasonable: aTox uses sane defaults, doesn't make guess about what you care about, and doesn't ignore what <b>you</b> want</li>
```

Reply:
Added, but reworded to fit under 120 cols. Thank you!